static_src/util/cf_api.js

```diff
@@ -14,6 +14,28 @@ import userActions from '../actions/user_actions.js';
 const APIV = '/v2';
 
 export default {
+  version: APIV,
+
+  fetch(url, action, multiple) {
```

Comment:
For a separate PR, we need to take care of pagination of the returned resources.